bin/dart_openapi_codegen.dart

```diff
@@ -16,7 +16,9 @@ String className(String s) => fixup(s
     .map((x) => x.capitalized)
     .join(''));
 
-String variableName(String s) => className(s).uncapitalized;
+// TODO: maybe make the keyword escape more robust later
+String variableName(String s) =>
+    s == 'override' ? 'overrideField' : className(s).uncapitalized;
```

Comment:
Since when is override a keyword?

Reply:
well you can't have a field name called override :D

Reply:
it could just be my linter actually brb

Reply:
ok no, you can't have @override and a field name called override in the same class

```
• lib/matrix_api_lite/generated/model.dart:2875:3
Annotation must be either a const reference or constructor invalid_annotation
          ```